src/platforms/dotnet/metrics/index.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Set Up Metrics
+description: "Learn how to set up Metrics."
+sidebar_order: 9000
+---
+
+<PlatformContent includePath="profiling/index/preface" />
+
+# Emit a counter
+
+Counters are the simplest types of metrics. With counters, you can count occurrences of certain events.
+
+To emit a counter, you can do as follows:
+```csharp
+// Incrementing a counter by one for each button click.
+SentrySdk.Metrics.Increment("ButtonClicked",
+    tags: new Dictionary<string, string> {{ "region", "us-west-1" }});
+```
+
+# Emit a set
+
+Sets are useful for counting unique occurrences. With sets, you can obtain the count of unique elements that you added to it, making them very useful for certain cases.
+
+To emit a set, you can do as follows:
+```csharp
+// Adding a set of unique occurrences.
+SentrySdk.Metrics.Set("CheckoutCount", 7,
+    unit: MeasurementUnit.Custom("items"),
+    tags: new Dictionary<string, string> {{ "region", "us-west-1" }});
+```
+
+# Emit a distribution
+
+Distributions are useful to get the most insights from your data. With distributions, you can obtain aggregations such as `p90`, `min`, `max`, and `avg` making them very powerful.
+
+To emit a distribution, you can do as follows:
```

Comment:
```suggestion
# Emit a Distribution

Distributions help you get the most insights from your data by allowing you to obtain aggregations such as `p90`, `min`, `max`, and `avg`.

To emit a distribution, do the following:
```
I'm not sure if there's a logical order to the different "emits", but it would be good to keep the order consistent for all the SDKs- right now, PHP has the following order: [Setup][Emit a counter][Emit a distribution][Emit a set]
[Emit a gauge].